test/compilerlib/CompilerLibTest.cpp

```diff
@@ -49,13 +57,28 @@ int main(int argc, char *argv[]) {
   if (outputBaseName == "") {
     outputBaseName = testFileName.substr(0, testFileName.find_last_of("."));
   }
+
+  OptLevel optLevel = OptLevel::O0;
+  bool hasOptLevel = true;
+  if (optO0)
+    optLevel = OptLevel::O0;
+  else if (optO1)
+    optLevel = OptLevel::O1;
+  else if (optO2)
+    optLevel = OptLevel::O2;
+  else if (optO3)
+    optLevel = OptLevel::O3;
+  else
+    hasOptLevel = false;
+
   int retVal = 0;
   if (compileFromFile) {
     const char *errorMessage = NULL;
     retVal = omCompileFromFile(testFileName.c_str(), outputBaseName.c_str(),
         onnx_mlir::EmitLib, mcpu.empty() ? nullptr : mcpu.c_str(),
         march.empty() ? nullptr : march.c_str(),
-        mtriple.empty() ? nullptr : mtriple.c_str(), &errorMessage);
+        mtriple.empty() ? nullptr : mtriple.c_str(),
+        hasOptLevel ? &optLevel : nullptr, &errorMessage);
```

Comment:
[minor]: we could just pass the optLevel here given that the default is `OptLevel::O0`. Doing that would avoid the need to define `hasOptLevel`.